manifests/init.pp

```diff
@@ -84,6 +87,7 @@
   Boolean $output_log_dropped_packets,
   Boolean $input_log_dropped_packets,
   Hash $rules,
+  Hash $chains,
```

Comment:
Missing default, or comment bellow `@param` is wrong.

Reply:
`$chains` is initialized in `data/common.yml`, just like `$rules` already is.